spec/db/models.rb

```diff
@@ -135,6 +135,8 @@ class Book < VirtualTotalTestBase
   has_many :photos, :as => :imageable, :class_name => "Photo"
   has_one :current_photo, -> { all.merge(Photo.order(:id => :desc)) }, :as => :imageable, :class_name => "Photo"
 
+  virtual_has_many :author_books, :through => :author, :source => :books
```

Comment:
Can we get another test for :through without source?  I assume that would be something like

```
  virtual_has_many :books, :through => :author
```

Reply:
Yea, having trouble thinking up this one. Especially without a source

A book has many books?

Reply:
I really didn't like it but...
Was better than anything I could come up with. - I added it with tests